templates/certificates/_accomplishment-footer.html

```diff
@@ -0,0 +1,16 @@
+<div class="wrapper-footer">
+
+    <footer class="footer-app" role="contentinfo" id="company-info">
+        <div class="footer-app-copyright">
+            <p class="copy copy-micro">&copy; <script>document.write(new Date().getFullYear())</script> Microsoft Corporation</p>
```

Comment:
nit: this will display the customer's currently set date. in other words, I think it's better to get the year from the server-side. i'm not going to block on this,